contracts/test/ocr2/proxy.test.ts

```diff
@@ -28,8 +29,6 @@ describe('proxy.cairo', function () {
     })
 
     console.log(proxy.address)
```

Comment:
is this console.log necessary still?

Reply:
It was already here, I 'm removing it

Reply:
ok, then let's leave these comment for Kristijan to see!